Adds 15.2 release notes (#81)
gitkraken/gitlens-docs: gitlens/gitlens-release-notes-current.md

---

<a id="v15-2"></a>

## Version 15.2

#### Wednesday, July 10th, 2024

GitLens 15.2 introduces several new features and improvements. This release focuses on enhancing the Commit Graph and Launchpad, expanding AI-generated messaging, and additional AI support for the latest Anthropic models.

<img src="/wp-content/uploads/gl-15-2-hero.png" class="img-responsive center img-bordered">

### Commit Graph Improvements

The Commit Graph now provides rich hovers on commit rows. These hovers include detailed commit information and links to related pull requests, issues, and linking to Inspect. We've also added pull request markers to the graph scroll and minimap, offering visual queues of PR activity in your repo history.

### Launchpad Improvements

> We've added a new "Open Worktree in New Window" action to the Launchpad. This feature allows you to open specific branches of pull requests in a separate VS Code window, minimizing context switching when working on multiple tasks. We've also added an "Open on GitHub" button to rows, enabling easy viewing on the GitHub website.

Reviewer: ```suggestion
We've added a new "Open Worktree in New Window" action to the Launchpad. This feature allows you to open specific branches of pull requests as a worktree in a separate VS Code window, minimizing context switching when working on multiple tasks.
```

The "open on GitHub" button being in a few other rows isn't really significant enough to merit mention here. You can already go to GitHub by clicking on the rows themselves, this is just a very minor convenience improvement for those who prefer to use the button.

Author: Thanks, good catch
